quinn/src/tests.rs

```diff
@@ -376,6 +376,14 @@ fn run_echo(client_addr: SocketAddr, server_addr: SocketAddr) {
 
         let handle = runtime.spawn(async move {
             let incoming = server_incoming.next().await.unwrap();
+
+            if cfg!(target_os = "linux") {
+                let local_ip = incoming.local_ip().expect("Local IP must be available");
+                assert!(local_ip.is_loopback());
+            } else {
+                assert_eq!(None, incoming.local_ip());
+            }
```

Comment:
Add a note here to update the getter docs in case the platform support falls out of sync?

Reply:
Here, or rather to the platform files? This might be the place where people realize that more changes are necessary - because CI fails. But I somehow don't think that "update doc comments" is best kept in the tests.  In the platform files - maybe. But if someone works on adding it to windows they won't look at the unix file.

Reply:
I mean, it could be both, but as you say, I'm not sure someone adding support for a different platform would review the code for unix. So I think adding it here would definitely be useful, maybe do the platform files as well (I didn't see a straightforward single location where it could be added, but I could easily be wrong about that).